### DISCUSSION TITLE
Missing hscroll_indicator from the released source code of 5.0.0 - Is there any reason?

### DISCUSSION THREAD

**Discussion by Roshkj** [ArthurSonzogni/FTXUI#779] [Q&A] [answered]
It seems to me that hscroll_indicator is deliberately removed from the released source code. Is there any reason, or was that added after version 5.0.0 is released? The file under the question is `scroll_indicator.cpp`.

**Comment by ArthurSonzogni** [comment, marked as answer]
See the changelog file:
https://github.com/ArthurSonzogni/FTXUI/blob/main/CHANGELOG.md

It was added after 5.0.0. It will be part of the next version.

**Reply by Roshkj** [reply]
Okay. Thank you.

Another question: Is there any plan for you to add the scroller module from your repo: https://github.com/ArthurSonzogni/git-tui/tree/master/src

That would be helpful for many I think.
